README.rst

```diff
@@ -118,6 +118,20 @@ there with access to these translations.
 Then follow `these instructions <https://docs.transifex.com/api/introduction#authentication>`_
 to get an API token, and set TRANSIFEX_API_TOKEN in your environment with its value.
 
+The cc-licenses-data repo should be cloned next to the cc-licenses repo. (It can
+be elsewhere, then you need to set TRANSLATION_REPOSITORY_DIRECTORY to its location.)
+Be sure to clone using a URLlike "git@github..." and not "https://github...", or you won't
+be able to push to it.
```

Comment:
Instead of saying a `URLlike` can we say something like "when cloning, make sure to copy the url under ssh that begins with `git@github...`. If you copy the https url, you will be unable to push."